oidc-api/src/main/java/uk/gov/di/authentication/oidc/validators/RequestObjectAuthorizeValidator.java

```diff
@@ -231,6 +249,11 @@ private Optional<AuthRequestError> validateVtr(JWTClaimsSet jwtClaimsSet, URI re
             return errorResponse(
                     redirectURI,
                     new ErrorObject(OAuth2Error.INVALID_REQUEST_CODE, "Request vtr not valid"));
+        } catch (ParseException e) {
```

Comment:
Could catch both in one block as they return the same thing, ie.
`catch (IllegalArgumentException | ParseException e)`

Reply:
I didn't go with this as there's more information in the `IllegalArgumentException` log that we don't have at the `ParseException` stage, and I thought it might make it unclear for people reading the logs